x-pack/plugins/searchprofiler/public/application/components/profile_query_editor/profile_query_editor.tsx

```diff
@@ -135,18 +148,40 @@ export const ProfileQueryEditor = memo(() => {
             <EuiSpacer size="s" />
           </EuiFlexItem>
           <EuiFlexItem grow={5}>
-            <EuiButton
-              data-test-subj="profileButton"
-              fill
-              disabled={!licenseEnabled}
-              onClick={() => handleProfileClick()}
-            >
-              <EuiText>
-                {i18n.translate('xpack.searchProfiler.formProfileButtonLabel', {
-                  defaultMessage: 'Profile',
+            {licenseEnabled && !hasIndices ? (
```

Comment:
Yup, I agree is odd. I did this way because the tooltip is only displayed when there is not indices. But I can think on a way of avoiding duplicate code.